Source/WebCore/html/HTMLMediaElement.h

```diff
@@ -496,7 +496,8 @@ class HTMLMediaElement
     WEBCORE_EXPORT static HashSet<SecurityOriginData> originsInMediaCache(const String&);
     WEBCORE_EXPORT static void clearMediaCache(const String&, WallTime modifiedSince = { });
     WEBCORE_EXPORT static void clearMediaCacheForOrigins(const String&, const HashSet<SecurityOriginData>&);
-    static void resetMediaEngines();
+
+    WEBCORE_EXPORT static void resetMediaEngines();
```

Comment:
Is It really necessary, cannot we just call platformStrategies()->mediaStrategy().resetMediaEngines() in the call sites?

Reply:
done